c/httpserver.c

```diff
@@ -5876,6 +5901,17 @@ void registerHttpServerModuleWithBase(HttpServer *server, STCBase *base)
 int mainHttpLoop(HttpServer *server){
   STCBase *base = server->base;
   /* server pointer will be copied/accessible from module->data */
+
+
+  zowelog(NULL, LOG_COMP_HTTPSERVER, ZOWE_LOG_INFO,
+    "ZSS5742I %s Create semaphore table for datasets\n", __FUNCTION__);
+
+  int i;
+  for(i=0; i<N_SEM_TABLE_ENTRIES; i++)
```

Comment:
Please use the style you see in the file you're changing. 
* Do you need `i` outside the for loop? If not, please don't declare it outside the for loop.
* Please throw in some blanks around the operators

Reply:
Yes, part of cleanup